aries-site/src/pages/components/header.mdx

```diff
@@ -6,8 +6,11 @@ import {
   HeaderNavigationExample,
   HeaderAvatarExample,
   HeaderSearchActionsExample,
+  PageHeaderIntroExample,
 } from '../../examples';
 
+Header is extremely versatile and can be built by mixing and matching a variety of components to provide users with relevant interactive elements to navigate through or engage with an application.
```

Comment:
relevant, interactive elements

[the comma geek in me spoke up] lol

Reply:
Fixed thanks.